arch/arm/boot/dts/overlays/pifi-dac-hd-overlay.dts

```diff
@@ -3,23 +3,23 @@
 /plugin/;
 
 / {
-	compatible = "brcm,bcm2708";
+	compatible = "brcm,bcm2735";
```

Comment:
Oooh - so close.

Reply:
Ouch.  Thank you, again, for your patience.